.github/workflows/_local_cd_release.yml

```diff
@@ -11,14 +11,16 @@ jobs:
     if: github.repository == 'SINTEF/ci-cd' && startsWith(github.ref, 'refs/tags/v')
     uses: ./.github/workflows/cd_release.yml
     with:
+      # General
       git_username: "TEAM 4.0[bot]"
       git_email: "TEAM4.0@SINTEF.no"
+      release_branch: main
+
+      # Distribution
       python_package: true
       publish_on_pypi: false
+      upload_distribution: false
```

Comment:
Should we not upload the distribution (publish) as part of this workflow?

Reply:
The `ci-cd` package itself is not published on PyPI. There is therefore no need to upload it as an artifact (where it will be removed in any case after 90 days by default).